### PR TITLE
[Merged by Bors] - chore: Missing `to_additive` on `CommSemigroup.isCentralScalar`

### PR DESCRIPTION
**Pull request by YaelDillies**
The SHA was already updated in #2498, but the `#align` was forgotten, and the instance name was incorrectly generated.

Match https://github.com/leanprover-community/mathlib/pull/16975.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]
bors merge

Thanks!

**Comment by eric-wieser** [comment]
bors d+

Did we misport something elsewhere? Or forget to forward-port something?

**Comment by YaelDillies** [comment]
Nothing misported, but `to_additive` is missing the translation `IsCentralScalar → IsCentralVAdd`. @fpvandoorn, can you explain how to do the translation? I tried something but it didn't work.

File: Mathlib/Tactic/ToAdditive.lean [hunk 1]
```diff
@@ -750,6 +750,7 @@ def nameDict : String → List String
 | "unit"        => ["add", "Unit"]
 | "units"       => ["add", "Units"]
 | "rootable"    => ["divisible"]
+| "isCentralScalar" => ["isCentralVAdd"]
```
Comment (eric-wieser) [inline Mathlib/Tactic/ToAdditive.lean:753]: This belongs in `fixAbbreviation` instead.

**Comment by fpvandoorn** [comment]
Thanks!

bors merge

**Pull request closed without merging**